developer-docs/latest/deployment/heroku.md

```diff
@@ -235,7 +238,25 @@ We also need to set the `NODE_ENV` variable on Heroku to `production` to ensure
 heroku config:set NODE_ENV=production
 ```
 
-#### 5. Install the `pg` node module
+#### 5. Create your Strapi server config for production
+
+Create a new `server.js` in a new [env](../concepts/configurations.html#environments) folder. In this file we only need one key, the `url`, to notify Strapi what our public Heroku domain is. All other settings will automatically be pulled from the default `./config/server.js`.
```

Comment:
Avoid using "we".
I would suggest "In this file you only need one key (...)" or "In this file only one key is needed (...)".